backend/packages/Upgrade/src/api/services/ExperimentService.ts

```diff
@@ -1134,6 +1138,16 @@ export class ExperimentService {
       });
   }
 
+  async updateStateTimeLogs(experimentDoc: Experiment, transactionalEntityManager: EntityManager) {
+    const stateTimeLogDoc = new StateTimeLog();
+    stateTimeLogDoc.id = uuid();
+    stateTimeLogDoc.fromState = EXPERIMENT_STATE.INACTIVE;
```

Comment:
You are hard-coding `fromState` here but it is getting called from `updateExperimentInDB`. What if the experiment is in a different state than `Inactive`.